zjb/templates/results.html.j2

```diff
@@ -36,4 +39,86 @@
                 </tfoot>
             </table>
             {%- endfor -%}
+
+            <script>
+                function openTextbox(project, branch, pipeline, job) {
+                    const textbox = document.createElement('textarea');
+                    textbox.setAttribute('id', 'textboxForNote');
```

Comment:
replacing those with modal dialog.  Converting the request to draft until the change i finished